engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/sink/BaseKafkaAvroSinkFactory.scala

```diff
@@ -33,7 +33,7 @@ abstract class BaseKafkaAvroSinkFactory extends SinkFactory {
     val schemaUsedInRuntime = schemaDeterminer.toRuntimeSchema(schemaData)
 
     val clientId = s"${processMetaData.id}-${preparedTopic.prepared}"
-    new KafkaAvroSink(preparedTopic, version, key, value, kafkaConfig, serializationSchemaFactory,
+    new KafkaAvroSink(preparedTopic, version, key, AvroSinkPrimitiveValue(value), kafkaConfig, serializationSchemaFactory,
```

Comment:
Is this really primitive value? Maybe we should change name to sth like SingleValueParameter?

Reply:
:+1: 
https://github.com/TouK/nussknacker/pull/1405/commits/9c6c765331d9c50413c687cc6f30b044aeeac7d8